src/act.informative.cpp

```diff
@@ -3710,8 +3710,24 @@ ACMD(do_cyberware)
 
   send_to_char("You have the following cyberware:\r\n", ch);
   for (obj = ch->cyberware; obj != NULL; obj = obj->next_content) {
-    snprintf(buf, sizeof(buf), "%-40s Essence: %0.2f\r\n",
-            GET_OBJ_NAME(obj), ((float)GET_OBJ_VAL(obj, 4) / 100));
+    if (GET_CYBERWARE_TYPE(obj) == CYB_FINGERTIP) {
+      if (obj->contains && GET_OBJ_TYPE(obj->contains) == ITEM_WEAPON && GET_HOLSTER_READY_STATUS(obj) ) {
+        snprintf(buf, sizeof(buf), "%-32s ^Y(W) (R)^n Essence: %0.2f\r\n", GET_OBJ_NAME(obj), ((float)GET_CYBERWARE_ESSENCE_COST(obj) / 100));
```

Comment:
It just occurred to me that this isn't screenreader-friendly; we can merge it as is and I can fix it to avoid you having to drop DB to switch branches again, but generally speaking, anything that includes abbreviations like this should have a separate display mode for anyone flagged PRF_SCREENREADER. Something like `<item name> (Contains Weapon) (Readied)  Essence: ...`